backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/excel/process/ExportToExcelProcess.java

```diff
@@ -0,0 +1,32 @@
+package de.metas.impexp.excel.process;
+
+import de.metas.impexp.spreadsheet.process.ExportToSpreadsheetProcess;
+
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2018 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+
+// FIXME: remote this class
+@Deprecated
+public class ExportToExcelProcess extends ExportToSpreadsheetProcess
```

Comment:
pls remove it

Reply:
also provide a migration script which updates AD_Process and sets Classname to ExportToSpreadsheetProcess where the classname is ExportToExcelProcess

Reply:
update: now i saw u already did: https://github.com/metasfresh/metasfresh/pull/11503/files#diff-e88039b5abbd6ed8c056b87f937c1a2968fb9af8575a490da069d777e29d3716R144

Reply:
removed